docs/how-to-guides/integrating-autoware/creating-your-autoware-meta-repository/creating-autoware-meta-repository.md

```diff
@@ -0,0 +1,57 @@
+# Creating Autoware meta-repository
+
+## What is Meta-repository?
+
+A meta-repository is a repository that manages multiple repositories, and Autoware is one of them.
+It serves as a centralized control point for referencing, configuring, and versioning other repositories.
+
+By using Ansible and vcs, you can automatically set up your Autoware.
+`autoware.repos` file manages the configuration of multiple repositories.
+
+Note: vcs stands for Version Control System, such as Git or Subversion.
+
+## How to create and customize your autoware meta-repository
+
+### 1. Fork autoware repository
```

Comment:
Above title is `create and customize`, fork is method that they create a repository , purpose is creating one, right?

Reply:
I was mixing up create and fork. 
 I would like to amend the `### 1. Fork autoware repository` to
```diff 
- ### Fork autoware repository 
- + ### Create autowaare repository. ```

Reply:
update https://github.com/autowarefoundation/autoware-documentation/pull/414/commits/18ef7d4612808f0e7213446cacf0f29b4a8f7ea7